core/src/main/java/org/infinispan/functional/impl/CounterConfigurationMetaParam.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.functional.impl;
```

Comment:
I'm not strongly opposed to this, as it does simplify things here, but do we want to be mixing counters with core?

Reply:
I will move it to commons. 
This class was in counters and I didn't want to create a cycle dependency between core <-> counters.

Reply:
well, it has to stay in core. `MetaParam.Writable` is only available here.